.github/workflows/build.yaml

```diff
@@ -71,8 +71,8 @@ jobs:
           run: |
             mkdir build
             cd build
-            cmake -DCMAKE_BUILD_TYPE=Release -GNinja ..
-            ninja
```

Comment:
If we're removing ninja we might want to remove `ninja-build` from dependencies in `.cicd/platforms/ubuntu*.Dockerfile`